xml/apache_os.xml

```diff
@@ -262,48 +295,56 @@
 
   <fingerprint pattern="\(White Box\)">
     <description>White Box Enterprise Linux</description>
+    <example>Apache/2.0.46 (White Box)</example>
     <param pos="0" name="os.vendor" value="White Box"/>
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Enterprise Linux"/>
   </fingerprint>
 
   <fingerprint pattern="\(UnitedLinux\)">
     <description>UnitedLinux</description>
+    <example>Apache/1.3.26 (UnitedLinux) mod_ssl/2.8.10</example>
     <param pos="0" name="os.vendor" value="UnitedLinux"/>
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
   </fingerprint>
 
   <fingerprint pattern="\(PLD/Linux\)">
     <description>PLD Linux</description>
+    <example>Apache/1.3.42 (PLD/Linux)</example>
     <param pos="0" name="os.vendor" value="PLD"/>
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
   </fingerprint>
 
   <fingerprint pattern="\(Vine/Linux\)">
     <description>Vine Linux</description>
+    <example>Apache/1.3.27 (Unix) (Vine/Linux)</example>
     <param pos="0" name="os.vendor" value="Vine"/>
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
   </fingerprint>
 
   <fingerprint pattern="\(rPath\)">
     <description>rPath Linux</description>
+    <example>Apache/2.2.9 (rPath)</example>
     <param pos="0" name="os.vendor" value="rPath"/>
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
   </fingerprint>
 
   <fingerprint pattern="\(StartCom Linux\)">
     <description>StartCom Linux</description>
+    <!-- Replace with a useful (real world) example if one is found -->
+    <example>Apache (StartCom Linux)</example>
```

Comment:
I couldn't find an example for this, so I just hand-crafted a string to satisfy the regular expression. StartCom appears to be defunct.

Reply:
I found a mirror that has ISOs for that distro! https://ftp.nluug.nl/pub/os/Linux/distr/startcom/

Reply:
good call!

```
$ wget -q https://ftp.nluug.nl/pub/os/Linux/distr/startcom/AS-5.0.0/os/x86_64/StartCom/RPMS/httpd-2.2.3-31.SEL5_4.x86_64.rpm && \
rpm2cpio httpd-2.2.3-31.SEL5_4.x86_64.rpm | cpio -ivdm ./usr/sbin/httpd && \
strings usr/sbin/httpd | grep StartCom && \
rm httpd-2.2.3-31.SEL5_4.x86_64.rpm
./usr/sbin/httpd
6466 blocks
Apache/2.2.3 (StartCom)
Apache/2.2.3 (StartCom) (Release 31.SEL5_4)
```

Reply:
```
$ URL="https://ftp.nluug.nl/pub/os/Linux/distr/startcom/ML-5.0.5/os/x86_64/StartCom/RPMS/httpd-2.2.0-5.1.2.x86_64.rpm" FNAME="`basename $URL`" bash -c 'wget -q $URL && rpm2cpio $FNAME | cpio -ivdm ./usr/sbin/httpd && strings usr/sbin/httpd | grep StartCom && rm $FNAME' 
./usr/sbin/httpd
14294 blocks
Apache/2.2.0 (StartCom Linux)
```